rotkehlchen/types.py

```diff
@@ -417,6 +417,15 @@ def get_chain_type(self) -> str:
         # else
         return 'eth2'  # the outlier
 
+    def get_name(self) -> str:
+        return SUPPORTED_BLOCKCHAIN_TO_NAME[self]
```

Comment:
you don't need this. Why are you adding a "name" here?

1. Was this needed by the frontend?
2. This is already happening by the `str()` method inherited by `SerializableEnumValueMixin2`. So you don't need a new method here nor an entire new mapping dictionary just to capitalize it. Why do they need capitalization?

Reply:
> Was this needed by the frontend?

yes, based on discussion in discord thread

> Why do they need capitalization?

i believe they need capitalization for display purposes here

![](https://media.discordapp.net/attachments/1062309439333470260/1062625458832814140/image.png)

but this can be handled on frontend and we can avoid the additonal mapping

Reply:
yes they should handle if if they need. Just use the str() method.

Reply:
updated